src/Handler/CurlMultiHandler.php

```diff
@@ -2,11 +2,27 @@
 namespace M6Web\Bundle\GuzzleHttpBundle\Handler;
 
 use GuzzleHttp\Handler\CurlMultiHandler as GuzzleCurlMultiHandler;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
- * extends guzzle CurlMultiHandler
+ * Extends guzzle CurlMultiHandler
  */
 class CurlMultiHandler extends GuzzleCurlMultiHandler
 {
+    /** @var EventDispatcherInterface Event Dispatcher */
+    protected $eventDispatcher;
+
+    public function __construct(EventDispatcherInterface $eventDispatcher, array $options)
+    {
+        $this->eventDispatcher = $eventDispatcher;
+
+        parent::__construct($options);
+    }
+
     use CacheTrait;
-}
\ No newline at end of file
+
+    public function getEventDispatcher(): EventDispatcherInterface
```

Comment:
trait should be above contruct or event atributes I think, same in CurlHandler.php